deploy.sh

```diff
@@ -9,6 +9,15 @@
 # graphic logo are (registered/a) trademark(s) of Plan International.
 set -e
 
+# Read environment variable file for the environment
```

Comment:
This enables us to store environment variables needed for deployment in a `.env.<environment>` file. Of course we should in most cases only deploy through Github Actions (especially as the num of configuration and variables increases),  but this helps a lot when you're setting up a new environment

Variables I currently have defined in my local file:
```
SLACK_WEBHOOK_URL=
KIBANA_USERNAME=
KIBANA_PASSWORD=
```